cli.js

```diff
@@ -139,6 +139,7 @@ var api = new Api({
 	babelConfig: conf.babel,
 	resolveTestsFrom: cli.input.length === 0 ? pkgDir : process.cwd(),
 	timeout: cli.flags.timeout,
+	pkgDir,
```

Comment:
You can't use object literal shorthand. We still support older Node.js versions.